src/client/Microsoft.Identity.Client/TokenCache.ITokenCacheInternal.cs

```diff
@@ -420,6 +428,18 @@ private static string GetPreferredUsernameFromIdToken(bool isAdfsAuthority, IdTo
 
             CacheRefreshReason cacheInfoTelemetry = CacheRefreshReason.NotApplicable;
 
+            if (AcquireTokenInLongRunningOboWasCalled(requestParams) && tokenCacheItems.Count == 0)
+            {
+                logger.Error("[FindAccessTokenAsync] AcquireTokenInLongRunningProcess was called and OBO token was not found in the cache.");
```

Comment:
Im wondering if it makes sense to indicate that the developer needs to call InitiateLongRunningProcess first?